src/main/java/spoon/support/visitor/equals/EqualsChecker.java

```diff
@@ -254,4 +255,33 @@ public void visitCtContinue(CtContinue e) {
 		}
 		super.visitCtLiteral(e);
 	}
+
+	@Override
+	public void visitCtComment(CtComment e) {
```

Comment:
But `visitCtComment` takes a `CtComment`. Should I really depend on the `CtCommentImpl` implementation?